visualizer/src/audio/util.hpp

```diff
@@ -36,28 +48,56 @@ namespace audio {
 
     /* Opens a portaudio stream with a given callback and data pointer */
     void portaudio_open_stream(PaStream** stream, PaStreamCallback* streamCallback, void* data) {
-        audio::guard_portaudio_error(
-            Pa_OpenDefaultStream(
-                /* The stream to open*/
-                stream,
-                /* No input channels */
-                0,
-                /* Stereo output */
-                2,
-                /* 32 bit floating point output */
-                paFloat32,
-                /* Sample rate of the output*/
-                audio::sample_rate,
-                /*  Frames per buffer, i.e. the number of sample frames that PortAudio will
-                 *  request from the callback. Many apps may want to use paFramesPerBufferUnspecified, which
-                 *  tells PortAudio to pick the best, possibly changing, buffer size.
-                 */
-                paFramesPerBufferUnspecified,
-                /* This is your callback function */
-                streamCallback,
-                /* This is a pointer that will be passed to your callback*/
-                data),
-            "audio::open_stream()");
+        PaDeviceIndex num_devices = Pa_GetDeviceCount();
+        PaDeviceIndex the_schiit_box = -1;
+
+        for (int i = 0; i < num_devices; i++) {
+            const PaDeviceInfo* device_info = Pa_GetDeviceInfo(i);
+            std::string device_name = device_info->name;
+            if (device_name.find("Schiit Modi 3") != std::string::npos) {
+                the_schiit_box = i;
+            }
+        }
```

Comment:
portaudio stopped being able to select the default stream for some reason idk... hard coded settings to the schiit box that we use instead